### ISSUE TITLE
LOCATION

### ISSUE THREAD

**Issue by eryk9** [Alovoa/alovoa#409]
Add the possibility of selecting the location in manual mode, as it automatically blocks.
It blocks when the operating system has restrictions of automatic locality, as the user does not give access to the location.
Consequently because Alovoa does not block must have a manual selection method set by the user
![Schermata del 2024-07-06 19-54-43](https://github.com/Alovoa/alovoa/assets/63969458/61e49669-bd7f-4403-b9ea-c147d82653c0)

**Comment by bobemoe** [comment]
Yeah I even granted permission and I still get this. Its a laptop so I have no GPS so not sure how it will figure my location but still no ketchup.
